esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,100 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note:** Before you follow these steps to add your recipe, you must be
+able to successfully run it with ESMValtool without error.
+
+* Run ESMValTool locally with your recipe, make sure to take note of the memory
+  and time expenditure provided by the terminal output::
+
+    esmvaltool run <your_recipe_name.yml>
+
+* Run the same recipe on JASMIN, again taking note of the memory and time
+  expenditure printed to the terminal.
+
+* Stop any running recipe_test_workflow workflows::
+
+    cylc stop "a_running_recipe_test_workflow"
+
+* Add the recipe to the ``[task parameters]`` section of the |RTW| ``flow.cylc``
+  workflow file (make sure to place it within the "fast" or "medium" category
+  depending on how long it took to run):
+
+* Run the workflow. The process task for the new recipe should succeed, but its
+  compare task should fail with an error that the reference data (KGO) does
+  not exist::
+
+    cylc vip -O <your_site_name>
+
+* Locate the workflow run folder of the workflow you just completed.
+
+* Copy the output files from the "cycle" folder (run/share/bin/cycle) of the
+  workflow you just ran, to your site specific KGO rootpath folder
+  (this folder should be set as the value for the variable "KGO_ROOT_PATH="
+  in your rose-suite<your_site>.conf file)::
+
+    cp -r <directory_of_recipe_output_cycle_folder> <KGO_rootpath_folder>
+
+* Change directory to the rootpath KGO directory::
+
+    cd <KGO_rootpath_directory>
+
+* Allow write permissions for all users on the directory and it's
+  subdirectories of the recipe you've added to the KGO folder::
+
+    chmod a+w <the_directory_of_the_recipe_you_have_copied_into_the_KGO_folder>
+
+* Run the RTW again::
+
+    cylc vip -O <your_site_name>
+
+* The workflow should now succeed.
+
+* Take note of how long the run took  to complete on cylc. This can be found in the `job.time` section of the task listed as `process_<your_recipe>`.
+
+* Locate your local <site>.cylc file, found in (recipe_test_workflow/site).
```

Comment:
So I've followed steps from here to fix the issue I got with the very first process task run.

Reply:
@chrisbillowsMO, can this comment be resolved? Is there anything needed here? 🤔

Reply:
No - it was just a note on the path/order I ended up following through the instructions as they were then. I'm sure that's moot now so I'll resolve.